lib/recipe/widgets/recipe_ingredients_screen.dart

```diff
@@ -131,10 +133,18 @@ class Ingredients extends StatelessWidget {
                   ),
                   child: Card(
                       child: ListTile(
-                    title: Text(state
-                        .recipes[recipeIndex].ingredients[index].name.value),
+                    title:
+                        (state.recipes[recipeIndex].ingredients[index].name ==
+                                "Other (please specify)")
+                            ? Text(state.recipes[recipeIndex].ingredients[index]
+                                    .otherName ??
+                                '')
+                            : Text(state.recipes[recipeIndex].ingredients[index]
+                                    .name ??
+                                ''),
```

Comment:
yeah, I think `customName` would be more expressive than `otherName` when compared to name.